CefSharp.Wpf/ChromiumWebBrowser.cs

```diff
@@ -1588,11 +1589,19 @@ private void WindowStateChanged(object sender, EventArgs e)
             } 
         }
 
+        private void updateBrowserScreenLocation()
+        {
+            if (PresentationSource.FromVisual(this) != null)
```

Comment:
This is a very expensive check to make (have a look at the `.Net` reference source if you want to know more, it's a few levels deep in the functions calls when it starts walking up the tree from memory).

`PresentationSourceChangedHandler` already tracks additions/removal if the `HwndSource`, you can probably check if `source != null` instead.